app/javascript/packs/src/settings/components/account/preferences/InputTimezone.jsx

```diff
@@ -0,0 +1,97 @@
+import React, { Component } from "react";
+import PropType from "prop-types";
+import { Button } from "react-bootstrap";
+import styled from "styled-components";
+import TimezonePicker from "react-bootstrap-timezone-picker";
+import "react-bootstrap-timezone-picker/dist/react-bootstrap-timezone-picker.min.css";
```

Comment:
hmm ... I don't like this :smile: check how webpacker include this or in a separate link/pack or bundles all together